packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -3065,55 +3129,86 @@ AND event_name = '${eventName}'`,
       (metric) => metric.sql === sqlQuery && metric.type === type
     );
   }
+  getAutoGeneratedFactTableSqlQuery(
+    eventName: string,
+    schemaFormat: AutoFactTableSchemas,
+    schema?: string
+  ): string {
+    const {
+      timestampColumn,
+      userIdColumn,
+      anonymousIdColumn,
+      getTrackedEventTablePath,
+      getEventFilterWhereClause,
+      filterColumns,
+      getDateLimitClause,
+    } = this.getSchemaFormatConfig(schemaFormat);
+
+    const sqlQuery = `
+      SELECT
+        ${userIdColumn} as user_id,
+        ${anonymousIdColumn} as anonymous_id,
+        ${timestampColumn} as timestamp,
+        ${filterColumns}
```

Comment:
This assumes `filterColumns` is not empty.  If it is empty, the trailing comma after `timestamp` will be a syntax error